src/server.h

```diff
@@ -2219,6 +2219,7 @@ struct valkeyServer {
     sds availability_zone; /* When run in a cloud environment we can configure the availability zone it is running in */
     /* Local environment */
     char *locale_collate;
+    char *debug_context; /* A free-form string that can be used by the user for any purpose. It has no impact on server behavior. */
```

Comment:
```suggestion
    /* A free-form string that will get printed out after a crash. It is guaranteed to never impact server behavior. */
    char *debug_context; 
```
Clang format will complain about this as well.